### PR TITLE
Optional Learn Spelling just like the others

### PR DESCRIPTION
**Pull request by iongion**
Added a flag just like for the others as it was missing.

As a mitigation, it can be disabled globally through electron `webPreferences` but that has other effects.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -120,19 +120,26 @@ The first argument is an array of default actions that can be used. The second a
...
+Default: `true`
+
+Show the `Learn Spelling {selection}` menu item when right-clicking text.
+
 #### showLookUpSelection
 
 Type: `boolean`\
 Default: `true`
 
-Show the `Look Up {selection}` menu item when right-clicking text on macOS.
```
Comment (iongion) [inline readme.md:128]: This works on Linux too

File: readme.md [hunk 2]
```diff
@@ -120,19 +120,26 @@ The first argument is an array of default actions that can be used. The second a
...
+Show the `Look Up {selection}` menu item when right-clicking text.
 
 #### showSearchWithGoogle
 
 Type: `boolean`\
 Default: `true`
 
-Show the `Search with Google` menu item when right-clicking text on macOS.
+Show the `Search with Google` menu item when right-clicking text.
```
Comment (sindresorhus) [inline readme.md:142]: You need to update the TS docs too.

File: readme.md [hunk 3]
```diff
@@ -120,19 +120,26 @@ The first argument is an array of default actions that can be used. The second a
 
 `MenuItem` labels may contain the placeholder `{selection}` which will be replaced by the currently selected text as described in [`options.labels`](#labels).
 
+#### showLearnSpelling
+
+Type: `boolean`\
+Default: `true`
+
+Show the `Learn Spelling {selection}` menu item when right-clicking text.
```
Comment (sindresorhus) [inline readme.md:128]: This makes it sound like it always shows, which is not true.

File: readme.md (hunk 2 above)
Comment (iongion) [reply readme.md:142]: Sure, sorry, where do I do that ?

File: readme.md (hunk 3 above)
Comment (iongion) [reply readme.md:128]: You do have the info

> To get spellchecking, “Correct Automatically”, and “Learn Spelling” in the menu, please enable the `spellcheck` preference in browser window: `new BrowserWindow({webPreferences: {spellcheck: true}})

I thought it would be redundant, could rephrase it to this ?

> Show the `Learn Spelling {selection}` menu item when right-clicking text. The `spellcheck` preference in browser window must be enabled.

File: readme.md (hunk 2 above)
Comment (iongion) [reply readme.md:142]: Found it! Updating!

File: readme.md (hunk 3 above)
Comment (sindresorhus) [reply readme.md]: I meant; it will only show when right-clicking misspelled words.

Comment (iongion) [reply readme.md]: Adjust with this to be precise ?

> Show the `Learn Spelling {selection}` menu item when right-clicking text. Even if `true`, the `spellcheck` preference in browser window must still be enabled. It will also only show when right-clicking misspelled words.
